docs/DeveloperGuide.adoc

```diff
@@ -800,7 +801,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new person to either Leads or Contacts|organise my sales targets better for future sales decisions
```

Comment:
'Add a new lead' don't add new person

Reply:
Splitting off into leads and contact sections probably works better. For formatting, there should be a space before each pipe.